src/bundle/Resources/views/themes/admin/ui/component/table/table.html.twig

```diff
@@ -13,47 +13,49 @@
 {% block between_header_and_table %}{% endblock %}
 
 {% block table %}
-    <table
-        class="ibexa-table table
-        {{ class|default('') -}}"
-        {{ html.attributes(attr|default({})) }}
-    >
-        <thead>
-            {% block thead %}
-                {% if head_cols is defined %}
-                    {% include '@ezdesign/ui/component/table/table_head_row.html.twig' with {
-                        head_cols: head_cols,
-                    } %}
-                {% endif %}
-            {% endblock %}
-        </thead>
-        <tbody
-            class="ibexa-table__body
-            {{ table_body_class|default('') -}}"
-            {{ html.attributes(table_body_attr|default({})) }}
+    <div class="ibexa-scrollable-wrapper">
```

Comment:
Do we want to make it an option?

Reply:
I think you didn't understand me. I was thinking about adding an option for 3rd party to disable with default value `true`.